ide/src/store.ts

```diff
@@ -148,7 +150,10 @@ function handleFirstActionableEffect(
   for (let i = 0; i < effectQueue.length; i += 1) {
     const effect = effectQueue[i];
 
+    // console.log('doing effect.effectKey ', effect.effectKey);
     switch (effect.effectKey) {
+      case 'startEditTimer':
```

Comment:
What's a summary of the effect this has? Is it just that this event was being fired from somewhere so it needed to be handled somehow? Is that still happening in Examplaritor?